model_zoo/PCLR/deployment/v1/pclr_model_schema.json

```
@@ -0,0 +1,16 @@
+{
+  "inputs": [ 
+    {
+      "name": "ecg_tensor",
```

```suggestion
      "name": "ecg",
```
@daniellepace looks like this should be ecg and not ecg_tensor